### PR TITLE
Deprecate/ugc post share content media

### LINKED ISSUE #13: [Bug] Adjust package to account for deprecated table `ugc_post_share_content_media`
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

`ugc_post_share_content_media` was deprecated [internally in a recent release of Linkedin Pages.](https://fivetran.com/docs/applications/linkedin-company-pages/changelog). This is causing errors for customers attempting to use Transformations on Linkedin Pages.

[Examining the new ERD](https://docs.google.com/presentation/d/1PbexBiOTxplv7TlmNZ7uNo8EDT2Lr5g0Ys_ealh4qzI/edit#slide=id.g311502b468_5_443), it's unclear if the staging models built on top of this source can still be utilized. We will likely need to scope whether this model can be modified or needs to be removed. 

### Relevant error log or model output

_No response_

### Expected behavior

Transformations should run.

### dbt Project configurations

standard to transformations: https://github.com/fivetran/dbt_linkedin_pages/blob/main/dbt_project.yml

### Package versions

packages:
  - package: fivetran/linkedin_pages
    version: [">=0.2.0", "<0.3.0"]

### What database are you using dbt with?

other (mention it in "Additional Context")

### dbt Version

What version transformations is running on.

### Additional Context

 https://fivetran.slack.com/archives/C02919TN9AT/p1695850062879859

### Are you willing to open a PR to help address this issue?

- [X] Yes.
- [ ] Yes, but I will need assistance and will schedule time during our [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance
- [ ] No.

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
## PR Overview
**This PR will address the following Issue/Feature:** #13 

**This PR will result in the following new package version:**

- v0.3.0

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

- This PR deprecates the `ugc_post_share_content_media` table and updates the `stg_linkedin_pages__ugc_post_history` table. The
 
### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [ ] ~dbt run –full-refresh~
- [x] dbt run
- [x] dbt test
- [ ] ~dbt run –vars (if applicable)~

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

See internal ticket for examples. 

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ ] ~DECISIONLOG updates have been updated (if applicable)~
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
📄

### CODE REVIEW

File: models/stg_linkedin_pages__post_content.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        poll_settings_duration,
+        poll_settings_is_voter_visible_to_author,
+        poll_settings_vote_selection_type,
+        poll_unique_voters_count,
+        type as post_type,
+        source_relation
+    from fields
+)
+
+select * from final
```
Comment (fivetran-joemarkiewicz) [inline models/stg_linkedin_pages__post_content.sql]: very small format suggestion
```suggestion
select * 
from final
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,31 @@
-# dbt_linkedin_pages_source v0.UPDATE.UPDATE
+# dbt_linkedin_pages_source v0.3.0
+[PR #14](https://github.com/fivetran/dbt_linkedin_pages_source/pull/14) includes the following breaking changes:
+## 🚨 Breaking Changes 🚨:
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:3]: Can we include that these changes were made as a result of a change in the upstream connector and then link the connector release notes here as well. So users are aware of the reason for these changes.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-catfritz this looks good! I just have a few suggestions to apply then this will be good to go!

File: CHANGELOG.md (hunk 2 above)
Comment (fivetran-catfritz) [reply CHANGELOG.md:3]: Updated!

**Pull request merged**